rps-game.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Rock, Paper, Scissors</title>
+    <link rel="stylesheet" type="text/css" href="./src/main.css">
+    <link rel="stylesheet" type="text/css" href="./src/rps-game.css">
+    <link href="https://fonts.googleapis.com/css?family=DM+Serif+Display&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <h1>Let's Play Rock, Paper, Scissors!</h1>
+        <nav>
+            <a href="./">Home</a> | 
+            <a href="guessing.html">Guessing Game</a> |    
+            <a href="rps-game.html">Rock, Paper, Scissors</a>
+        </nav>
+    </header>
+    <main>
+        <section id="rules-section">
+            <h2>The Rules</h2>
+            <p>Ready to play? Enter your bet and make your pick. In case you need a refresher, here are the rules:</p>
+            <ul>
+                <li>Rock crushes scissors.</li>
+                <li>Scissors cut paper.</li>
+                <li>Paper covers rock.</li>
+            </ul>
+            
+        </section>
+        <section id="status-section">  
+            <h2>Your Status</h2> 
+            <table>
+                <tr>
+                    <td colspan="2"><p>Your Balance: $<span id="user-balance">500</span></p></td>
+                </tr>
+                <tr>
+                    <td>Wins: </td>
+                    <td>Losses:</td>
+                </tr>
+                <tr>
+                    <td><span id="win-count">0</span></td>
+                    <td><span id="loss-count">0</span></td>
+                </tr>
+            </table>
+
+        </section>
+        <section class="game-section">
+        <h3>Your Bet: </h3>
```

Comment:
make sure to indent everything inside this section one tab over, as they are all children of this section